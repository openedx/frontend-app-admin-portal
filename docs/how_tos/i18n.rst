####################
React App i18n HOWTO
####################

This document has moved to the frontend-platform repo: https://github.com/openedx/frontend-platform/blob/master/docs/how_tos/i18n.rst
