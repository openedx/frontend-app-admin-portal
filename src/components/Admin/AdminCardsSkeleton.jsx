import React from 'react';
import { Skeleton } from '@edx/paragon';

const AdminCardsSkeleton = () => (
  <div
    className="admin-cards-skeleton mb-3 d-md-flex w-100"
  >
    <div className="sr-only">Loading...</div>
    <Skeleton />
    <Skeleton />
    <Skeleton />
    <Skeleton />
  </div>
);

export default AdminCardsSkeleton;
