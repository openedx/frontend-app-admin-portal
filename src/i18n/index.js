import { messages as paragonMessages } from '@edx/paragon';

export default [
  paragonMessages,
];
