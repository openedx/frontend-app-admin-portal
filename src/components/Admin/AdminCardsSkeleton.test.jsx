import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from '@edx/frontend-platform/i18n';
import AdminCardsSkeleton from './AdminCardsSkeleton';

describe('AdminCardsSkeleton', () => {
  it('renders a skeleton', () => {
    const tree = renderer
      .create((
        <IntlProvider>
          <AdminCardsSkeleton />
        </IntlProvider>
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
