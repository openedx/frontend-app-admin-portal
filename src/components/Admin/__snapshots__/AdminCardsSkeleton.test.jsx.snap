// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminCardsSkeleton renders a skeleton 1`] = `
<div
  className="admin-cards-skeleton mb-3 d-md-flex w-100"
>
  <div
    className="sr-only"
  >
    Loading...
  </div>
  <span
    aria-busy={true}
    aria-live="polite"
  >
    <span
      className="react-loading-skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <br />
  </span>
  <span
    aria-busy={true}
    aria-live="polite"
  >
    <span
      className="react-loading-skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <br />
  </span>
  <span
    aria-busy={true}
    aria-live="polite"
  >
    <span
      className="react-loading-skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <br />
  </span>
  <span
    aria-busy={true}
    aria-live="polite"
  >
    <span
      className="react-loading-skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <br />
  </span>
</div>
`;
